concept/type/attributetype.feature

```diff
@@ -11,83 +11,76 @@ Feature: Concept Attribute Type
     Given connection has been opened
     Given connection does not have any database
     Given connection create database: typedb
-    Given connection opens schema transaction for database: typedb
+    Given connection open schema transaction for database: typedb
+
+    # TODO: Create structs in concept api
+    Given put struct type: custom-struct
+    Given struct(custom-struct) create field: custom-field
+    Given struct(custom-struct) get field(custom-field); set value-type: string
+
+    Given transaction commits
+    Given connection open schema transaction for database: typedb
+
+########################
+# attribute type common
+########################
 
   Scenario: Root attribute type cannot be deleted
     Then delete attribute type: attribute; fails
 
-  Scenario: Attribute types can be created
+  Scenario Outline: Attribute types can be created with <value-type> value type
```

Comment:
The issue with `Scenario Outlines` is that we reset the database for each of the `Examples`, it is a fully new step. 

I like the idea of testing different set ups with different value types (so all the main actions are covered by multiple value types), but at the same time, it will increase the test runtime. Feel free to stop me if I overdid the Examples (especially relevant for `attributetype.feature` and `owns.feature` because of the value types!).